docs/src/main/sphinx/installation/cli.rst

```diff
@@ -358,8 +364,51 @@ The following table list the available options for Kerberos authentication:
   * - ``--krb5-service-principal-pattern``
     - Remote kerberos service principal pattern (default: ${SERVICE}@${HOST})
 
-See :doc:`/security/cli` for more information on configuring and using Kerberos
-with the CLI.
+Troubleshooting
+~~~~~~~~~~~~~~~
+
+Many of the same steps, that can be used when troubleshooting the :ref:`Trino
```

Comment:
Where does this link to?

Reply:
It links to the troubleshooting section of the Kerberos Authentication doc.